### PR TITLE
Small documentation fixups

### PR DESCRIPTION
**Pull request by aanm**

### CODE REVIEW

File: Documentation/network/servicemesh/mutual-authentication/installation.rst [hunk 1]
```diff
@@ -9,6 +9,25 @@ Installation
 
 .. tabs::
 
+    .. group-tab:: Cilium CLI
+
+        .. include:: ../../../installation/cli-download.rst
+
+        Mutual authentication and associated SPIRE server can be enabled with the following command.
+        Note this requires the Cilium CLI Helm mode (version 0.15 or later).
```
Comment (zacharysarah) [inline Documentation/network/servicemesh/mutual-authentication/installation.rst]: Use active voice, specific is better than general, punctuation nits

I'm not attached to "its" re:SPIRE server, feel free to replace with another article like "an"
```suggestion
        You can enable mutual authentication and its associated SPIRE server with the following command.
        This command requires the Cilium CLI Helm mode version 0.15 or later.
```

File: Documentation/network/servicemesh/mutual-authentication/installation.rst [hunk 2]
```diff
@@ -9,6 +9,25 @@ Installation
...
+
+        .. code-block:: shell-session
+
+            $ cilium install \
+                --helm-set authentication.mutual.spire.enabled=true \
+                --helm-set authentication.mutual.spire.install.enabled=true
+
+        Next you can check the status of the Cilium agent and operator:
```
Comment (zacharysarah) [inline Documentation/network/servicemesh/mutual-authentication/installation.rst]: Good use of "you" to address the user, but we can trim it here
```suggestion
        Next, check the status of the Cilium agent and operator:
```

**Review by zacharysarah — DISMISSED** [review]
@aanm Nice. ✨ This review includes changes to text you haven't touched, but it's a chance to make small but meaningful improvements in a related PR.

**Review by meyskens — APPROVED** [review]
For technical part of sig-servicemesh LGTM

**Pull request merged**